x-pack/plugins/infra/public/pages/metrics/hosts/hosts_content.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Query, TimeRange } from '@kbn/es-query';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { i18n } from '@kbn/i18n';
+import React, { useState, useCallback } from 'react';
+import { SearchBar } from '@kbn/unified-search-plugin/public';
+import { InfraLoadingPanel } from '../../../components/loading';
+import { useMetricsDataViewContext } from './hooks/use_data_view';
+import { HostsTable } from './components/hosts_list_table';
+import { InfraClientStartDeps } from '../../../types';
+
+export const HostsContent: React.FunctionComponent = () => {
+  const {
+    services: { data },
+  } = useKibana<InfraClientStartDeps>();
+  const [dateRange, setDateRange] = useState<TimeRange>({ from: 'now-15m', to: 'now' });
```

Comment:
Should we look to use the global Kibana timerange service here instead?

Reply:
I'm getting the time range from the Unified Search component and storing it as state so I can pass it to the lens component.  Would it make sense to use that here?  Could you point me to an example?